docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debezium/connect:latest
+
+USER root
+
+RUN microdnf -y install git java-11-openjdk-devel
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
+ENV KAFKA_HOME /kafka
```

Comment:
This is already [inherited from a parent image](https://github.com/debezium/container-images/blob/28d4950cebd799051f22dd852f11793b49ec52bb/kafka/2.4/Dockerfile#L17); can we remove it?